src/helpers/ethereumAuthenticate.js

```diff
@@ -0,0 +1,34 @@
+const ethers = require('ethers')
+
+async function authenticateFromConfig(ethereumConfig, log = console.info) {
+    let wallet
+    let provider
+
+    if (ethereumConfig.url) {
```

Comment:
Connection to blockchain network shouldn't be necessary for resolving address from private key though?